tests/extendrtests/DESCRIPTION

```diff
@@ -1,23 +1,25 @@
 Package: extendrtests
 Title: Integration Tests for Calling Extendr from an R Package
-Version: 0.1.0
+Version: 0.2.0
 Authors@R: c(
     person("Andy", "Thomason", , "andy@andythomason.com", c("aut")),
     person("Claus O.", "Wilke", , "wilke@austin.utexas.edu", c("aut", "cre"),
-      comment = c(ORCID = "0000-0002-7470-9261"))
+      comment = c(ORCID = "0000-0002-7470-9261")),
+    person("Ilia", "Kosenkov", , "ilia.kosenkov@outlook.com", c("aut"), comment = c(ORCID = "0000-0001-5563-7840"))
   )
 Description:
-  This package provides tests for extendr functionality used from
+  Provides tests for extendr functionality used from
```

Comment:
These R changes seem unrelated to the PR?

Reply:
I cleaned a little bit integration test. I can of course take it out into a separate PR if you feel like it should be done.

Reply:
Ok, I moved these bits to #447. As soon as it is merged, I'll drop some of the commits from this branch.

Reply:
Once it's merged this part of the diff should disappear right?